js/turtledefs.js

```diff
@@ -531,7 +531,7 @@ function createHelpContent() {
         'dockblock': [_('The Dock block block connections two blocks.'), 'documentation', ''],
         'runblock': [_('The Run block block runs a block. It accepts two types of arguments: block number or block name.'), 'documentation', ''],
         'moveblock': [_('The Move block block moves a block.'), 'documentation', ''],
-        'deleteblock': [_('The Delete block block removes a block.'), 'documentation', ''],
+        'deleteblock': [_('The Delete block block removes a block.'), 'documentation', '', 'deletehelp'],
```

Comment:
Did you test this? It looks wrong. Should it be 'deleteblock': [_('The Delete block block removes a block.'), 'documentation', 'deletehelp'],

Reply:
Yes, I tested it. I think the third element in the array is the name of the SVG then the fourth one is the name of the macro